src/contracts/middleware/BLSPubkeyRegistry.sol

```diff
@@ -0,0 +1,211 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity =0.8.12;
+
+
+import "../interfaces/IBLSPubkeyRegistry.sol";
+import "../interfaces/IRegistryCoordinator.sol";
+import "../interfaces/IBLSPublicKeyCompendium.sol";
+
+import "../libraries/BN254.sol";
+
+import "forge-std/Test.sol";
+
+
+contract BLSPubkeyRegistry is IBLSPubkeyRegistry, Test {
+
+    bytes32 internal constant ZERO_PK_HASH = hex"ad3228b676f7d3cd4284a5443f17f1962b36e491b30a40b2405849e597ba5fb5";
```

Comment:
comments on these variables would be nice. a good amount can be replicated from the interface